exonum/src/events/internal.rs

```diff
@@ -70,6 +91,20 @@ impl InternalPart {
                         }).map_err(|_| panic!("Can't execute shutdown"));
                         to_box(f)
                     }
+                    InternalRequest::VerifyTx(tx) => {
+                        if !txs_in_verification.insert(tx.raw().hash()) {
```

Comment:
IHMO straight order will be more intuitive: at first process situation, when we can insert transaction, then when we can't.
Is this negation required here?

Reply:
You're right, it's not really required